examples/plot_example_workflow.py

```diff
@@ -0,0 +1,151 @@
+"""
+===========================================================
+Example ``autoreject`` preprocessing workflow including ICA
+===========================================================
+
+This example demonstrates how to visualize data when preprocessing
+with :mod:`autoreject` and discusses decisions about when and which
+other preprocessing steps to use in combination.
+"""
+
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#         Mainak Jas <mjas@mgh.harvard.edu>
+#         Apoorva Karekal <apoorvak@uoregon.edu>
+#
+# License: BSD (3-clause)
+
+# sphinx_gallery_thumbnail_number = 2
+
+# %%
+# First, we download resting-state EEG data from a Parkinson's patient
+# from OpenNeuro. We will do this using ``openneuro-py`` which can be
+# installed with the command ``pip install openneuro-py``.
+
+import os
+import os.path as op
+import matplotlib.pyplot as plt
+import openneuro
+
+import mne
+import autoreject
+
+dataset = 'ds002778'  # The id code on OpenNeuro for this example dataset
+subject_id = 'pd14'
+
+target_dir = os.path.join(
+    os.path.dirname(autoreject.__file__), '..', 'examples', dataset)
+if not os.path.isdir(target_dir):
+    os.makedirs(target_dir)
+
+openneuro.download(dataset=dataset, target_dir=target_dir,
+                   include=[f'sub-{subject_id}/ses-off'])
+
+# %%
+# We will now load in the raw data from the bdf file downloaded from OpenNeuro
+# and, since this is resting-state data without any events, make regularly
+# spaced events with which to epoch the raw data. In evoked plot (the plot of
+# the average of the epochs) we can see that there may be some eyeblink
+# artifact contamination but, overall, the data is typical of
+# resting-state EEG.
+
+raw = mne.io.read_raw_bdf(op.join(target_dir, f'sub-{subject_id}',
+                                  'ses-off', 'eeg',
+                                  'sub-pd14_ses-off_task-rest_eeg.bdf'))
+raw.drop_channels(['EXG1', 'EXG2', 'EXG3', 'EXG4', 'EXG5',
```

Comment:
Would it not be better / more explicit to do `pick_types` instead?

Reply:
Unfortunately, they are not typed properly